other/verify_image gcpkms.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: verify-image-gcpkms
+  annotations:
+    policies.kyverno.io/title: Verify Image
```

Comment:
The title annotation is what the website filter will scrape, so these names should be unique.

Reply:
And there's already that name [here](https://kyverno.io/policies/other/verify_image/).

Reply:
Thank you so much, fixed.